RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHGC_cfi.py

```diff
@@ -21,21 +21,33 @@
              name = cms.string("PFHGCEERecHitCreator"),
              src  = cms.InputTag("HGCalRecHit:HGCEERecHits"),
              geometryInstance = cms.string("HGCalEESensitive"),
-             qualityTests = cms.VPSet(  
+             qualityTests = cms.VPSet(
+#                 cms.PSet(
+#                     name = cms.string("PFRecHitQTestHGCalThresholdSNR"),
+#                     thresholdSNR = cms.double(5.0),
+#                    ),
                 )              
            ),
            cms.PSet(
              name = cms.string("PFHGCHEFRecHitCreator"),
              src  = cms.InputTag("HGCalRecHit:HGCHEFRecHits"),
              geometryInstance = cms.string("HGCalHESiliconSensitive"),
              qualityTests = cms.VPSet(
+#                 cms.PSet(
+#                     name = cms.string("PFRecHitQTestHGCalThresholdSNR"),
+#                     thresholdSNR = cms.double(5.0),
+#                    ),                 
                 )
            ),
            cms.PSet(
              name = cms.string("PFHGCHEBRecHitCreator"),
              src  = cms.InputTag("HGCalRecHit:HGCHEBRecHits"),
              geometryInstance = cms.string(""),
              qualityTests = cms.VPSet(
+#                 cms.PSet(
```

Comment:
Indeed...  Even if it is just supposed to stay as such only temporarily, better to have a line explaining what the commented out config is meant to, for documentation purposes.
Removed the "+1" while waiting for those comments